scripts/emulated-performance.ps1

```diff
@@ -128,21 +131,50 @@ if ($IsWindows) {
     $QuicPerf = Join-Path $RootDir "/artifacts/bin/linux/$($Arch)_$($Config)_$($Tls)/quicperf"
 }
 
+# Path to the netput exectuable.
+$NetPut = "C:\Windows\System32\netput.exe"
```

Comment:
Shouldn't this be in PATH by default, so just `netput.exe` is accessible? Windows isn't required to be on the C drive, even though it usually is.

Reply:
It didn't work without it.

Reply:
Huh, thats weird. I wonder if System32 isn't in PATH on those systems. Oh well not a big deal.